### PR TITLE
docs: add schema-sync blog

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/blog/database-schema-synchronization-in-5-mins.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+feature_image: /static/blog/database-schema-synchronization-in-5-mins/sync-schema.webp
+tags: Education
+description: This article briefly describes the general scenarios of database schema synchronization and how to use this feature smoothly in Bytebase with pure UI operations.
+---
+
+This article briefly describes the general scenarios of **database schema synchronization** and how to use this feature smoothly in Bytebase with **pure UI operations**.
+
+## General Scenarios
+
+Bytebase's database **schema synchronization** feature (introduced in [v1.8.0](/changelog/bytebase-1-8-0)） supports copying a specific version of schema from one database to another. Without this feature, developers have to write SQL statements cautiously and apply them manually; Now with this feature, they can simply tell Bytebase which database schema needs to be replicated to the target database, and the whole process can be done through UI operations. This feature can solve the following scenarios:
```
Comment (tianzhou) [inline content/blog/database-schema-synchronization-in-5-mins.md:14]:
```suggestion
Bytebase's database **schema synchronization** feature (introduced in [1.8.0](/changelog/bytebase-1-8-0)） supports copying a specific schema version from one database to another. Without this feature, developers have to write SQL statements cautiously and apply them manually; Now with this feature, they can simply tell Bytebase which database schema needs to be replicated to the target database, and the whole process can be done from UI. This feature can solve the following scenarios:
```

File: content/blog/database-schema-synchronization-in-5-mins.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+### Scenario 1
+During the product release process, for different environments, such as dev / staging / prod, you need to do schema synchronization. For example, the R&D release process from dev to staging, then to prod in the forward direction, or from prod to dev for testing purposes in the reverse direction.
+
+### Scenario 2
+For the production environment, databases with identical schemas, such as SaaS, multi-region game deployment, etc.
+
+1) Manual modification by humans due to special ad-hoc reasons - causes database schema drift, which needs to renormalize.
```
Comment (tianzhou) [inline content/blog/database-schema-synchronization-in-5-mins.md:22]:
```suggestion
1) Manual modification by humans due to special ad-hoc reasons - causes database schema drift, which needs reconciliation.
```

File: content/blog/database-schema-synchronization-in-5-mins.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+2) The vendor maintains multiple versions of the product, so it also needs to normalize the schema.
```
Comment (tianzhou) [inline content/blog/database-schema-synchronization-in-5-mins.md:23]:
```suggestion
2) The vendor maintains multiple product versions, so it also needs to reconcile the schema.
```

File: content/blog/database-schema-synchronization-in-5-mins.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+2. Click **Preview Issue**, you will go to the new issue page with preview:
+![issue SQL](/static/blog/database-schema-synchronization-in-5-mins/issue-sql.webp)
+
+3. Click **Create**, and the **Issue** is truly created. **Approve** and **Fix** the issue, the schema synchronization will be completed.
+![dbprod t1](/static/blog/database-schema-synchronization-in-5-mins/dbprod-t1.webp)
+
+## Enterprise Plan
+With **Free** or **Team Plan**, you can only choose the latest version of history; Only with **Enterprise Plan** can you freely choose from the full migration history.
+
+From Bytebase [v1.8.0](/changelog/bytebase-1-8-0), you can run a 14-day **Team Plan** trial without hub registration. While having **Team Plan** trial, you can upgrade to the **Enterprise Plan** trial by clicking **Upgrade** in a specific feature.
```
Comment (tianzhou) [inline content/blog/database-schema-synchronization-in-5-mins.md:114]:
```suggestion
From Bytebase [1.8.0](/changelog/bytebase-1-8-0), you can run a 14-day **Team Plan** trial without hub registration. While having **Team Plan** trial, you can upgrade to the **Enterprise Plan** trial by clicking **Upgrade** in a specific feature.
```

File: content/blog/database-schema-synchronization-in-5-mins.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+![sync-schema UI](/static/blog/database-schema-synchronization-in-5-mins/sync-schema-ui.webp)
+
+2. Click **Preview Issue**, you will go to the new issue page with preview:
+![issue SQL](/static/blog/database-schema-synchronization-in-5-mins/issue-sql.webp)
+
+3. Click **Create**, and the **Issue** is truly created. **Approve** and **Fix** the issue, the schema synchronization will be completed.
+![dbprod t1](/static/blog/database-schema-synchronization-in-5-mins/dbprod-t1.webp)
+
+## Enterprise Plan
+With **Free** or **Team Plan**, you can only choose the latest version of history; Only with **Enterprise Plan** can you freely choose from the full migration history.
```
Comment (tianzhou) [inline content/blog/database-schema-synchronization-in-5-mins.md:112]:
```suggestion
In Bytebase **Free** or **Team Plan**, you can choose the latest history version; In **Enterprise Plan** , you can choose an arbitrary schema version from the full migration history.
```

File: content/blog/database-schema-synchronization-in-5-mins.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
+---
+title: Database Schema Synchronization in 5 mins
```
Comment (tianzhou) [inline content/blog/database-schema-synchronization-in-5-mins.md:2]: How to synchronize database schemas

and let's change the URL accordingly.

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml [hunk 7]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```
Comment (boojack) [inline pnpm-lock.yaml:1]: Unexpected change

**Comment by adela-bytebase** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**